openapi-cli/src/main/java/io/ballerina/openapi/cmd/utils/CodegenUtils.java

```diff
@@ -16,15 +16,19 @@
 
 package io.ballerina.openapi.cmd.utils;
 
+import java.io.File;
 import java.io.IOException;
 import java.io.PrintWriter;
+import java.nio.file.Files;
 import java.nio.file.Path;
+import java.nio.file.Paths;
 
 /**
  * Utilities used by ballerina openapi code generator.
  */
 public class CodegenUtils {
-
+    private static String licenseHeader = "";
```

Comment:
What is the purpose of this variable? Is there any getter/setter method for this?